proto/tpm.proto

```diff
@@ -47,8 +50,24 @@ message Pcrs {
   map<uint32, bytes> pcrs = 2;
 }
 
+// Information uniquely identifying a GCE instnace. Can be used to create an
```

Comment:
s/instnace/instance/ (also in PR description).

Reply:
Bahhhhh, thank you

Reply:
Oh also s/Attestaion/Attestation in the PR title (in case that gets used as the squashed commit message lol).

Reply:
Lol, should be fixed now